Formula/kubecfg.rb

```diff
@@ -22,7 +22,7 @@ def install
     cd "src/github.com/kubecfg/kubecfg" do
       system "make", "VERSION=v#{version}"
       bin.install "kubecfg"
-      pkgshare.install Dir["examples/*"], "testdata/kubecfg_test.jsonnet"
+      pkgshare.install Dir["examples/*", "testdata/*"]
```

Comment:
I'd rather not use `Dir` here because you're not operating on directories.

If you'd rather do something simpler:
```suggestion
      pkgshare.install (Pathname.pwd/"examples").children
      pkgshare.install (Pathname.pwd/"testdata").children
```
The mistake in my last suggestion was that these directories aren't in `buildpath` -- I missed the `cd` above. Apologies.